Sprint-3/slideshow/slideshow.js

```diff
@@ -6,5 +6,42 @@ const images = [
     "./assets/codingDeskSetup5.jpg"   
 ];
 
+let slideIndex = 0;
+const imgElement = document.getElementById('carousel-img');    
+const dots = document.getElementsByClassName('dot'); 
+
+function showSlide(index) {
+    if (index >= images.length) {
+        slideIndex = 0;
+    } else if (index < 0) {
+        slideIndex = images.length - 1;
+    } else {
+        slideIndex = index;
+    }
+    imgElement.src = images[slideIndex];
+    updateDots()
+}
+
+function updateDots() {
+    for (let i = 0; i < dots.length; i++) {
+        dots[i].className = dots[i].className.replace(" active", "");
+    }
+    dots[slideIndex].className += " active";
+}
+
+document.querySelector('.prev').addEventListener('click', () => {
```

Comment:
It is  a common practice that while selecting your html elements, it is better to organize them at the beginning of your script. You defined your img element at the beginning, also define other elements at the same place.